modules/ocf_www/files/build-vhosts

```diff
@@ -147,7 +162,7 @@ def report(*args, **kwargs):
 
 
 def build_config(src_vhosts, template, dev_config=False):
-    vhosts = set()
+    vhosts = list()
```

Comment:
I'm curious, why does this need changing from a `set` to a `list`? Are there duplicates now?

Reply:
Now that we added a dict the tuple isn't hashable. From a cursory reading (also I tested it), it doesn't seem to cause problems although we might be in hot water if we accidentally add duplicate declarations. Not exactly sure how nginx will react either.

Reply:
Ah I see, didn't realize that was a restriction of sets. Anyway, having this be a list seems reasonable to me, was just curious why this was needed.

Could maybe store the flags or metadata in another list or something instead as a workaround, but that might be over the top.